client/lxd_images.go

```diff
@@ -1052,7 +1052,7 @@ func (r *ProtocolLXD) ExportImage(fingerprint string, image api.ImageExportPost)
 	}
 
 	// Send the request
-	op, _, err := r.queryOperation("POST", "/images/"+url.PathEscape(fingerprint)+"/export", image, "", true)
+	op, _, err := r.queryOperation("POST", "/images/"+url.PathEscape(fingerprint)+"/export", &image, "", true)
```

Comment:
@simondeziel please can you add a test for this as looks like we have a gap that allowed this regression through, ta.

Reply:
@tomponline could you help me with how to test this out? I can hit the `GET` handler with `lxc image export <fingerprint>` but I still haven't found a way to get to the `POST` handler :/

Reply:
Which function is the post handler?

Reply:
`ExportImage()` from `client/lxd_images.go`